dist/adguard.json

```diff
@@ -48,6 +48,12 @@
             "url": "https://crashlytics.com/",
             "companyId": null
         },
+        "showrss": {
+            "name": "showRSS",
+            "categoryId": 8,
+            "url": "https://showrss.info/",
+            "companyId": "showRSS"
```

Comment:
Not `showrss`?

Reply:
> Not `showrss` like in https://github.com/AdguardTeam/companiesdb/pull/37/files#diff-58dc3c00cb661de579a63bdf6202d21dc2676f89c5e2faa04149d6421f55fc88R164?

`companyId` should be `showrss` I'd say. `name` should be able to stay as is based off other database entries.